ui/app/AppLayouts/Communities/helpers/PermissionsHelpers.qml

```diff
@@ -83,13 +83,19 @@ QtObject {
         }
     }
 
+    // OWNER AND TMASTER TOKENS related helpers:
+    readonly property string ownerTokenNameTag: "Owner-"
+    readonly property string tMasterTokenNameTag: "TMaster-"
+    readonly property string ownerTokenSymbolTag: "OWN"
+    readonly property string tMasterTokenSymbolTag: "TM"
+
     // It generates a symbol from a given community name.
     // It will be used to autogenerate the Owner and Token Master token symbols.
     function autogenerateSymbol(isOwner, communityName) {
```

Comment:
> communityNameToSymbol

We've agreed with @endulab that probably we will share this fuction with the backend since the backend will need to do something similar in some point!

Reply:
> How collisions are handled? Isn't it a problem that many communities may have tokens with the same symbol easily?

They are not handled and this has already been discussed with `John` that yes, there is the possibility of having communities with the same symbol but we cannot prevent it easily. So, at least right now, it will be not covered.